docs/recommendations.md

```diff
@@ -1,18 +1,139 @@
 # Recommendations
 
-This document lists recommendations from the Security SIG for
-the consideration of OpenTelemetry SIGs.
+This document lists recommendations from the Security SIG for the consideration of OpenTelemetry SIGs.
 
 ## CodeQL Scanning
 
-The organization uses [CodeQL](https://codeql.github.com/) for semantic analysis
-of the code in various repositories. The recommendation is to run CodeQL on every
-pull request and on every commit to the main branch.
+The organization uses [CodeQL](https://codeql.github.com/) for semantic analysis of the code in various repositories.
+The recommendation is to run CodeQL on every pull request and on every commit to the main branch.
 
 Issue: ([#15](https://github.com/open-telemetry/sig-security/issues/15))
 
+Note: we also have access to [Snyk](https://snyk.io) and you can add it to your repositories.
+
 ## Binding to Network Interfaces
-Always bind to localhost rather than to 0.0.0.0 or any interface, unless there is a specific need to do otherwise. Binding to localhost reduces the attack surface of your application by making it only accessible to devices on the same machine.
-Binding to 0.0.0.0 or "all" interfaces can make your application respond on all current and future network interfaces.
+
+Always bind to localhost rather than to 0.0.0.0 or any interface, unless there is a specific need to do otherwise.
+Binding to localhost reduces the attack surface of your application by making it only accessible to devices on the same
+machine. Binding to 0.0.0.0 or "all" interfaces can make your application respond on all current and future network
+interfaces.
 
 Issue: ([#19](https://github.com/open-telemetry/sig-security/issues/19#issue-1926445623))
+
+## Code signing / Provenance
+
+We recommend using [`sigstore`](https://www.sigstore.dev/) to create signatures for your SIG's deliverables, such as
+binaries, packages, and containers. In most cases, you can achieve that with GitHub's [build provenance attestation
+action](https://github.com/actions/attest-build-provenance).
+
+Example of actions for signing packages and binaries
+
+```yaml
```

Comment:
I built a working example as part of this PR: https://github.com/jpkrohling/otel-sig-security-example-go

1. Should it be part of this repository?
2. Should it be another repo, inside this organization?
3. Should we just add a link from here to that repo, under my account?

Reply:
I think that can be part of documentation and then link the repo for a real example, and that should be move to the org


One detail is that we generate the attestation, but we are not signing the images/packages; with this, we only sign the attestation.

maybe is also interesting to sign the artifacts too

Reply:
Alright, for simplicity, I'll keep the example external to this org for the moment and have a separate discussion for it.